.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] "
+labels: bug
+assignees: ''
```

Comment:
Maybe also give a list? "Leave blank unless you want to suggest specifically @rbooth200, @jeffjennings and/or @mtazzari."

Reply:
Good idea!
```suggestion
assignees: '' [Leave blank unless you want to suggest specifically @rbooth200, @jeffjennings and/or @mtazzari.]
```